src/Application/GameLuaBindings.cpp

```diff
@@ -188,6 +244,139 @@ void _registerSerializationBindings(sol::state_view &luaState, sol::table &table
+        "Water", CHARACTER_SKILL_WATER,
+        "Earth", CHARACTER_SKILL_EARTH,
+        "Spirit", CHARACTER_SKILL_SPIRIT,
+        "Mind", CHARACTER_SKILL_MIND,
+        "Body", CHARACTER_SKILL_BODY,
+        "Light", CHARACTER_SKILL_LIGHT,
+        "Dark", CHARACTER_SKILL_DARK,
+        "Item_ID", CHARACTER_SKILL_ITEM_ID,
+        "Merchant", CHARACTER_SKILL_MERCHANT,
+        "Repair", CHARACTER_SKILL_REPAIR,
+        "Bodybuilding", CHARACTER_SKILL_BODYBUILDING,
+        "Meditation", CHARACTER_SKILL_MEDITATION,
+        "Perception", CHARACTER_SKILL_PERCEPTION,
+        "Diplomacy", CHARACTER_SKILL_DIPLOMACY,
+        "Thievery", CHARACTER_SKILL_THIEVERY,
+        "Trap_Disarm", CHARACTER_SKILL_TRAP_DISARM,
+        "Dodge", CHARACTER_SKILL_DODGE,
+        "Unarmed", CHARACTER_SKILL_UNARMED,
+        "Monster_ID", CHARACTER_SKILL_MONSTER_ID,
+        "Armsmaster", CHARACTER_SKILL_ARMSMASTER,
+        "Stealing", CHARACTER_SKILL_STEALING,
+        "Alchemy", CHARACTER_SKILL_ALCHEMY,
+        "Learning", CHARACTER_SKILL_LEARNING,
+        "Club", CHARACTER_SKILL_CLUB,
+        "Misc", CHARACTER_SKILL_MISC
+    );
+
+    table.new_enum<false>("SkillMastery",
+        "None", CHARACTER_SKILL_MASTERY_NONE,
+        "Novice", CHARACTER_SKILL_MASTERY_NOVICE,
+        "Expert", CHARACTER_SKILL_MASTERY_EXPERT,
+        "Master", CHARACTER_SKILL_MASTERY_MASTER,
+        "Grandmaster", CHARACTER_SKILL_MASTERY_GRANDMASTER
+    );
+
+    table.new_enum<false>("ClassType",
+        "Knight", CLASS_KNIGHT,
+        "Cavalier", CLASS_CAVALIER,
+        "Champion", CLASS_CHAMPION,
+        "BlackKnight", CLASS_BLACK_KNIGHT,
+        "Thief", CLASS_THIEF,
+        "Rogue", CLASS_ROGUE,
+        "Spy", CLASS_SPY,
+        "Assassin", CLASS_ASSASSIN,
+        "Monk", CLASS_MONK,
+        "Initiate", CLASS_INITIATE,
+        "Master", CLASS_MASTER,
+        "Ninja", CLASS_NINJA,
+        "Paladin", CLASS_PALADIN,
+        "Crusader", CLASS_CRUSADER,
+        "Hero", CLASS_HERO,
+        "Villain", CLASS_VILLIAN,
+        "Archer", CLASS_ARCHER,
+        "WarriorMage", CLASS_WARRIOR_MAGE,
+        "MasterArcher", CLASS_MASTER_ARCHER,
+        "Sniper", CLASS_SNIPER,
+        "Ranger", CLASS_RANGER,
+        "Hunter", CLASS_HUNTER,
+        "RangerLord", CLASS_RANGER_LORD,
+        "BountyHunter", CLASS_BOUNTY_HUNTER,
+        "Cleric", CLASS_CLERIC,
+        "Priest", CLASS_PRIEST,
+        "PriestOfSun", CLASS_PRIEST_OF_SUN,
+        "PriestOfMoon", CLASS_PRIEST_OF_MOON,
+        "Druid", CLASS_DRUID,
+        "GreatDruid", CLASS_GREAT_DRUID,
+        "ArchDruid", CLASS_ARCH_DRUID,
+        "Warlock", CLASS_WARLOCK,
+        "Sorcerer", CLASS_SORCERER,
+        "Wizard", CLASS_WIZARD,
+        "Archmage", CLASS_ARCHAMGE,
+        "Lich", CLASS_LICH
+    );
+
+    // Let's not expose all the item types for now. I feel like it's too early.
+    table.new_enum<false>("ItemType",
+        "LichJarFull", ITEM_QUEST_LICH_JAR_FULL
+    );
```

Comment:
Regarding the enum names – this is OK for now, but I think ultimately we should have the same names for enums in lua and in C++. So this one will be `"ITEM_QUEST_LICH_JAR_FULL"`.